src/routes/routes.tsx

```diff
@@ -83,6 +84,7 @@ export const AppRoutes = () => {
       <Route path={paths.circles} element={<CirclesPage />} />
       <Route path={paths.createCircle} element={<CreateCirclePage />} />
       <Route path={paths.developers} element={<DevPortalPage />} />
+      <Route path={paths.discordLink} element={<DiscordPage />} />
```

Comment:
Let's make this only navigable in a dev environment for now.

Reply:
A feature flag would work too

Reply:
I like the idea of a feature flag, added the `discord` feature flag with https://github.com/coordinape/coordinape/pull/1625/commits/c41b0f7d6de2bebb96088df13aef67269b3983ca